bin/xud

```diff
@@ -87,6 +87,16 @@ const { argv } = require('yargs')
       type: 'boolean',
       default: undefined,
     },
+    'p2p.discover': {
+      describe: 'Do not send GET_NODES if false, when connecting to a peer',
+      type: 'boolean',
+      default: false,
+    },
+    'p2p.detectExternalIp': {
+      describe: 'Turn on/off discovering and sharing current external ip address on startup',
```

Comment:
How about "Detect and advertise external ip address to peers"?

Reply:
Whatever you say :)

Reply:
So if `detectexternalip` is false we will not advertise out ip? sound unrelated. 
We should allow the user to tell us which ip+port to use for advertising and separately from this option.

Reply:
We'll still advertise any addresses that are specified manually as you described.